src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -647,6 +651,9 @@ public static BigDecimal toScaledBigDecimal(final String value, final int scale,
      * @return Number created from the string (or null if the input is null)
      * @throws NumberFormatException if the value cannot be converted
      */
+    @SuppressWarnings("argument.type.incompatible")/*
```

Comment:
Please always use the checker prefix in the suppress warnings key. If somebody does a case study with a different checker, they won't want their warnings suppressed.